### PR TITLE
Adds sensible ravager keybind defaults

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

Please more than feel free to comment your own keybindings or suggest tweaks, see the file tab for added defaults

## Why It's Good For The Game

For ease of entering for new players into the caste

## Changelog
:cl:
qol: Added ravager keybind defaults
/:cl:

### CODE REVIEW

**Comment by Lefunny** [comment]
image

**Comment by yyzsong** [comment]
Either change it to select-ravage or make ravage into something that isn't Q. Ravs have to drag bodies too! (And select-ravage is way better)

File: code/datums/keybinding/xeno.dm [hunk 1]
```diff
@@ -548,18 +548,21 @@
...
 	description = "For the next few moments you will not go into crit and become resistant to explosives and immune to stagger and slowdown, but you still die if you take damage exceeding your crit health."
 	keybind_signal = COMSIG_XENOABILITY_ENDURE
+	hotkey_keys = list("F")
 
 /datum/keybinding/xeno/ravager_rage
 	name = "ravager_rage"
 	full_name = "Ravager: Rage"
 	description = "While active, you will temporarily recover plasma and sunder and gain a bonus to speed and melee damage in proportion to the percentage of your missing health. At negative HP your ability cooldowns reset and your slash damage restores health."
 	keybind_signal = COMSIG_XENOABILITY_RAGE
+	hotkey_keys = list("R")
```
Comment (TiviPlus) [inline code/datums/keybinding/xeno.dm:565]:
```suggestion
	hotkey_keys = list("Space")
```

File: code/datums/keybinding/xeno.dm [hunk 2]
```diff
@@ -572,6 +575,7 @@
 	full_name = "Ravager: Ravage"
 	description = ""
 	keybind_signal = COMSIG_XENOABILITY_RAVAGE
+	hotkey_keys = list("Q")
```
Comment (TiviPlus) [inline code/datums/keybinding/xeno.dm:578]:
```suggestion
	hotkey_keys = list("R")
```

**Pull request merged**